### ISSUE TITLE
Promotional video 

### ISSUE THREAD

**Issue by prettyvulture** [tcfev/forDem#114]
![image](https://user-images.githubusercontent.com/113349647/203425250-888c3180-fefd-4398-9307-322446a27ecd.png)
![image](https://user-images.githubusercontent.com/113349647/203429934-ccb5aeae-df73-43dc-9c67-f4b1618efb28.png)
![image](https://user-images.githubusercontent.com/113349647/203430055-108d8ca4-b456-4df1-bee1-a3d1006db270.png)
![image](https://user-images.githubusercontent.com/113349647/203430223-945b5774-3be1-4164-bb7b-c7e35d92a4fb.png)

- Get the promo video done.

[ForDem Media Timeline.pdf.pdf](https://github.com/tcfev/forDem/files/10071432/ForDem.Media.Timeline.pdf.pdf)

- Macro to-do's plan for docus' of TCFev / forDem

**Comment by prettyvulture** [comment]
- Video campaign idea - (Open for suggestions) 1:minute video idea, ideal for social media, tv ads, website feature video and web ad campaigns.

**A cute old man, practices ticking on an empty paper everyday. The visual seems mysterious for first few days, however, it is revealed that the old man is finally able to vote and is practicing ticking on the ballot paper. forDem is making it possbile  for those who were ineligible to vote for ages.**

**Comment by desputinski** [comment]
Good idea, but I think we have to be careful that no misunderstandings arise, as forDem will not be an official voting app for regular elections, like a voting machine in the US or something like that. But rather a tool for civil society...

**Comment by prettyvulture** [comment]
Yeah true. At the end of the video, our logo will popup with a message for donation, that message will tell how our app functions and benefits the marginalized.

**Comment by desputinski** [comment]
I have another idea for a promotion video etc. 
Maybe it would be good to have something like a quickly passing slide show with political action around the world. So just 0,5 second or so, one photo after another. For example
1. Protests in Iran / 2. Climate protests in Europe / 3. Protests against Oil Pipelines in the US / 4. Protests in China against zero covid policy / and so on.
Maybe even mixed with historical protests such as 1968 against the Vietnam war, student protests in Paris and so on....

**Comment by desputinski** [comment]
Also see: https://archive.org/details/fordem_202301
